pkg/gpu/probe.go

```diff
@@ -32,6 +32,7 @@ import (
 
 const (
 	gpuAttacherName = "gpu"
+	gpuModuleName   = gpuAttacherName
```

Comment:
i think it would be better to use [config.GPUMonitoringModule](https://github.com/DataDog/datadog-agent/blob/main/cmd/system-probe/config/config.go#L46)

(actually i can change all "magic string" references in probe.go in a separate PR)

Reply:
unfortunately, no, as it will cause cyclic import
we need to move [config.GPUMonitoringModule](https://github.com/DataDog/datadog-agent/blob/main/cmd/system-probe/config/config.go#L46) to be in `pkg/`